common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -138,6 +141,53 @@ public static void validateOrg(Request request) {
     }
   }
 
+
+  public static void validateCreateFirstRootOrg(Request request) {
+    if (StringUtils.isBlank((String) request.getRequest().get(JsonKey.ORG_NAME))) {
+      throw new ProjectCommonException(
+          ResponseCode.organisationNameRequired.getErrorCode(),
+          ResponseCode.organisationNameRequired.getErrorMessage(),
+          ERROR_CODE);
+    }
+    if (StringUtils.isEmpty((String) request.getRequest().get(JsonKey.CHANNEL))) {
+      throw new ProjectCommonException(
+          ResponseCode.channelIdRequiredForRootOrg.getErrorCode(),
+          ResponseCode.channelIdRequiredForRootOrg.getErrorMessage(),
+          ResponseCode.CLIENT_ERROR.getResponseCode());
+    } 
+    if(!validateRemoteAddress(request.get("remoteAddress").toString()))
+      {
+        throw new ProjectCommonException(
+          ResponseCode.requestNotAllowed.getErrorCode(),
+          ResponseCode.requestNotAllowed.getErrorMessage(),
+          ResponseCode.UNAUTHORIZED.getResponseCode());
+    } 
+        
+  }
+
+/**
+ * This method will validate the incoming request and validates the client host 
+ */
+  public static Boolean validateRemoteAddress(String remoteAddress){
+    Boolean validAddress = false;
+    try {
+      // compare client remote address with available hosts or localhost
+    if(remoteAddress.equals(ProjectUtil.getConfigValue(JsonKey.MW_SYSTEM_HOST)) || 
```

Comment:
Format code. (Run scripts/setup.sh in each sunbird_* repo. This formats code before each commit.).